images/build/go-runner/Dockerfile

```diff
@@ -49,10 +50,19 @@ RUN go env
 RUN go build -ldflags '-s -w -buildid= -extldflags "-static"' \
     -o go-runner ${package}
 
+RUN if [ "${IMGNAME}" = "go-runner-setcap" ]; then \
+      apt-get update && \
+      apt-get install -y -q --no-install-recommends --no-install-suggests --fix-missing gcc git libc6-dev make && \
+      git clone git://git.kernel.org/pub/scm/linux/kernel/git/morgan/libcap.git && \
+      cd libcap/progs && \
+      make; \
+    else mkdir -p libcap/progs; \
+    fi
```

Comment:
this looks strange... why wouldn't this be a layer on top of the go-runner dockerfile?

Reply:
I considered that but using go-runner to build this image would mean this wouldn't be a variant of the go-runner image (so we can't build them in parallel). But maybe its better to build go-runner-setcap, using go-runner as the base image?

Reply:
@justaugustus - WDYT? Is it better to build go-runner-setcap using go-runner as base much like the debian-iptables is build from debian-base?

Reply:
I have updated the PR based on the conversation I had in the sig-release slack regarding this PR. So resolving this conversation.